web/profiles/joinup/joinup.behat.inc

```diff
@@ -171,6 +174,53 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
   }
 
   /**
+   * Checks that the given select field has the given options.
+   *
+   * To do this we extract the text of the field which is the values of the
+   * select box concatenated. Then we check if the value is in this string.
+   *
+   * @param string $select
+   *    The name of the field element.
+   * @param string $options
+   *    The options to be checked separated by comma. Only labels are accepted.
+   *
+   * @throws Exception
+   *    Throws an exception when a value does not exists.
+   *
+   * @Given (the ):select field has the :options option(s)
+   */
+  public function checkSelectOption($select, $options) {
```

Comment:
Since the parameter itself is `$options`, the name should reflect that: `checkSelectOptions`.